SUBMISSION_GUIDELINES.md

```diff
@@ -0,0 +1,17 @@
+# Fiveable Open Source Project Submission Guidelines
+
+- Projects must be constructive and unique, repos that are intended to serve as a personal website or documentation websites are not allowed. The exception to this are external guides for libraries that don't provide a guide. 
+- Projects must have a clear CODE_OF_CONDUCT.md that clearly denounces any discriminatory, unwelcoming, or any otherwise harmful behavior that damages/hinders the positive environment we've strived to create.
```

Comment:
Should we link an example code of conduct? https://www.contributor-covenant.org/version/2/0/code_of_conduct/ perhaps